images/kubefed/Dockerfile

```diff
@@ -15,11 +15,13 @@
 
 FROM alpine:latest
 RUN apk --no-cache add ca-certificates
+RUN adduser -D hyperfed
 
-WORKDIR /root/
+WORKDIR /hyperfed/
```

Comment:
Should this be /home/hyperfed ?  Or should this directory be created and owned by the hyperfood user?

Reply:
directory should be owned by hyperfed user. updated docker file with `chown`